test/hotspot/jtreg/runtime/interpreter/LastJsrTest.java

```diff
@@ -24,17 +24,22 @@
 /*
  * @test
  * @bug 8335664
+ * @bug 8338924
  * @summary Ensure a program that ends with a JSR does not crash
  * @library /test/lib
  * @compile LastJsr.jasm
  * @compile LastJsrReachable.jasm
- * @run main/othervm LastJsrTest
+ * @run main/othervm
+ *      -Xbatch
+ *      LastJsrTest
```

Comment:
I suggest to keep old `command` too.
And I don't like spliting new command into several lines when it is very short.